src/common/components/DatePicker/DatePicker.tsx

```diff
@@ -7,7 +7,8 @@ import { Input } from '../Inputs'
 import CalendarIcon from 'common/assets/icons/ico-calendar.svg'
 import ChevronIcon from 'common/assets/icons/ico-chevron.svg'
 
-import './DatePicker.module.css'
+import 'react-datepicker/dist/react-datepicker.css'
```

Comment:
Can we import this CSS here?  https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/index.css (Line 30)